src/scss/global.scss

```diff
@@ -20,7 +20,7 @@ body {
 
 .main-content {
   contain: content; // see https://www.w3.org/TR/2018/CR-css-contain-1-20181108/#valdef-contain-content
-  padding-top: var(--main-content-pad-top);
+  padding-block-start: var(--main-content-pad-top);
```

Comment:
Is this orthogonal to the PR? Does this have an effect with different languages? (I can't think of any bottom-to-top languages, but I'm sure they exist?)

Reply:
you could take it out :shrug: